pkgs/misc/dart-sass-embedded/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, system
+, stdenvNoCC
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "dart-sass-embedded";
+  version = "1.62.0";
```

```suggestion
  version = "1.62.1";
```

Has been released yesterday :^)
https://github.com/sass/dart-sass-embedded/releases/tag/1.62.1